src/lib/replication/repl_dev/raft_repl_dev.cpp

```diff
@@ -836,6 +905,17 @@ void RaftReplDev::handle_error(repl_req_ptr_t const& rreq, ReplServiceError err)
     rreq->clear();
 }
 
+void RaftReplDev::replace_member(repl_req_ptr_t rreq) {
+    auto members = r_cast< const replace_members_ctx* >(rreq->header().cbytes());
+    replica_id_t member_in, member_out;
+    std::copy(members->out.begin(), members->out.end(), member_out.begin());
+    std::copy(members->in.begin(), members->in.end(), member_in.begin());
+    RD_LOGI("Raft repl replace_member member_out={} member_in={}", boost::uuids::to_string(member_out),
+            boost::uuids::to_string(member_in));
+
+    m_listener->replace_member(member_out, member_in);
```

Comment:
homeobject or listener can use this to update their pg metablks to remove old and add new member

Reply:
will revisit here if we need a separate add_member and remove_member